package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "vui-table",
+  "version": "1.0.0",
+  "description": "Mixins and CSS for applying table styles",
+  "scripts": {
+    "autoprefix": "postcss -c postcss.config.json",
```

Comment:
Is the auto prefixing necessary in this repo?  The CSS being produced here is just for the test, correct?  I think BSI (and any other consumer) should do its own auto prefixing.